cfme/tests/automate/test_git_import.py

```diff
@@ -79,26 +76,26 @@ def test_automate_git_domain_displayed_in_service(appliance, imported_domain):
         initialEstimate: 1/20h
         tags: automate
     """
+    url = "https://github.com/ramrexx/CloudForms_Essentials.git"
+    repo = AutomateGitRepository(url=url, verify_ssl=True, appliance=appliance)
+    imported_domain = repo.import_domain_from(branch="origin/master")
     collection = appliance.collections.catalog_items
     cat_item = collection.instantiate(collection.GENERIC, "test")
     view = navigate_to(cat_item, "Add")
-    view.provisioning_entry_point.click()
-    view.modal.tree.click_path(
+    path = (
         "Datastore",
         "{0} ({1}) ({0}) (Locked)".format(imported_domain.name, "origin/master"),
         "Service",
-        "Provisioning",
+        "Generic",
         "StateMachines",
-        "ServiceProvision_Template",
-        "CatalogItemInitialization"
+        "GenericLifecycle",
+        "provision"
     )
-    view.modal.include_domain.fill(True)
-    view.modal.apply.click()
-    assert view.provisioning_entry_point.value == ("/{}/Service/Provisioning/StateMachines/"
-        "ServiceProvision_Template/CatalogItemInitialization".format(imported_domain.name))
+    view.provisioning_entry_point.fill(path, include_domain=True)
+    assert view.provisioning_entry_point.value == (
+        "/CloudForms_Essentials/Service/Generic/StateMachines/GenericLifecycle/provision")
```

Comment:
Looks like you should still be formatting `imported_domain.name` into this expected string, instead of hard-coding `'CloudForms_Essentials'` into this string.

Reply:
Consider picking items from `path` and using `'/'.join([imported_domain.name, path[2], path[3], path[4], path[5], path[6]])` - but note this won't produce a string with a leading `'/'`

Reply:
Can we go with this @mshriver ?
```
assert view.provisioning_entry_point.value == (
        "/{}".format("/".join([imported_domain.name, path[2], path[3], path[4], path[5], path[6]]))
    )
```